tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/media_storage.py

```diff
@@ -10,26 +10,31 @@
 class MediaStorage(Base):
 
     _page_locator = (By.ID, 'mediaStorage')
-    _internal_storage_header_locator = (By.CSS_SELECTOR, '[data-l10n-id="storage-name-internal"]')
-    _external_storage_header_locator = (By.CSS_SELECTOR, '[data-l10n-id="storage-name-external-0"]')
     _advanced_header_locator = (By.CSS_SELECTOR, '[data-l10n-id="advanced"]')
-    _music_size_locator = (By.CSS_SELECTOR, '.color-music > a > .size')
-    _pictures_size_locator = (By.CSS_SELECTOR, '.color-pictures > a > .size')
-    _movies_size_locator = (By.CSS_SELECTOR, '.color-videos > a > .size')
 
-    _format_internal_selector_locator = (By.CSS_SELECTOR, '[data-l10n-id="format-sdcard-internal"]')
-    _format_sd_selector_locator = (By.CSS_SELECTOR, '[data-l10n-id="format-sdcard-external-0"]')
+    _internal_storage_header_locator = (By.CSS_SELECTOR, '[data-l10n-id="storage-name-internal"]')
```

Comment:
This duplication is the sign that we can improve things. Let's make a PageRegion, like we do [with the StatusBar](https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py#L30).

We can do:

``` python
class MediaStorage(Base):
   _internal_storage_locator = (By.CSS_SELECTOR, '[data-id="internal"] + ul')
    @property
    def internal_storage(self):
        element = self.marionette.find_element(*self._internal_storage_locator)
        return StorageRegion(self.marionette, element)
    
    # Same thing for external storage

    class StorageRegion(PageRegion):
        _music_size_locator = (By.CSS_SELECTOR, '[data-l10n-id="music-space"] + [data-l10n-id="storageSize"]')

        @property
        def music_size(self):
            return self.marionette.find_element(*self._music_size_locator).text

        # Same thing for movies, pictures and all
```